scalaj-client-circe/src/test/scala/endpoints/scalaj/client/CirceEntitiesTest.scala

```diff
@@ -0,0 +1,30 @@
+package endpoints.scalaj.client
+
+import endpoints.algebra.CirceEntities.CirceCodec
+import endpoints.testsuite.JsonTestApi
+import endpoints.testsuite.User
+import endpoints.testsuite.Address
+import endpoints.testsuite.client.JsonTestSuite
+import org.scalatest.WordSpec
+
+class TestClient(val address: String ) extends JsonTestApi with Endpoints with CirceEntities {
+
+  import io.circe.generic.auto._
+
+  override implicit def userCodec: CirceCodec[User] = CirceCodec.fromEncoderAndDecoder[User]
+
+  override implicit def addresCodec: CirceCodec[Address] = CirceCodec.fromEncoderAndDecoder[Address]
```

Comment:
Yeah, we have discussed that once in the past, and I even remembered at some point to remove it(it is automatically add by intellij) but then I forgot. Fixed